infra/firefly-core.yaml

```diff
@@ -0,0 +1,48 @@
+admin:
+  address: 0.0.0.0
+  enabled: true
+  port: 5101
+  publicURL: http://127.0.0.1:5101
+debug:
+  port: 6060
+event:
+  dbevents:
+    bufferSize: 10000
+http:
+  address: 0.0.0.0
+  port: 5000
+  publicURL: http://127.0.0.1:5000
+log:
+  level: debug
+metrics: {}
+namespaces:
+  default: default
+  predefined:
+  - defaultKey: addr_test1vqhkukz0285zvk0xrwk9jlq0075tx6furuzcjvzpnhtgelsuhhqc4
```

Comment:
Is there a way to inject this value from the `ADDR_FROM` environment variable?

Reply:
Probably. I was going through firefly helm chart and did not see any firefly config file, so probably it is configured via env variables. Is this a requirement?

Reply:
let's see if it causes issues in practice before spending too much time on it